modules/nwtc-library/src/NWTC_IO.f90

```diff
@@ -736,20 +686,17 @@ SUBROUTINE AllRPAry2 (  Ary, AryDim1, AryDim2, Descr, ErrStat, ErrMsg )
    ELSE
       ErrStat = ErrID_None
       ErrMsg  = ''
+      Ary = 0
```

Comment:
I moved this as we could get a memory issue if allocation didn't occur, but we try and zero it anyhow.